webrtc-stats.html

```diff
@@ -1860,6 +1879,84 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="mediasourcestats-dict*">
+        <h3>
+          RTCMediaSourceStats dictionary
+        </h3>
+        <p>
+          The <dfn>RTCMediaSourceStats</dfn> dictionary represents a local track that is attached
+          to a sender. It contains information about media sources such as frame rate and
+          resolution pre-encoding. This is in contrast to <code>RTCOutboundRtpStreamStats</code>
+          whose members describe transmission and post-encoding metrics. For example, a track may
+          be captured from a high-resolution camera but be transmitted in low resolution due to
+          CPU and network conditions.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCMediaSourceStats : RTCStats {
+             unsigned long   width;
+             unsigned long   height;
+             unsigned long   frames;
+             unsigned long   framesPerSecond;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCMediaSourceStats</a> Members
+            </h2>
+            <dl data-link-for="RTCMediaSourceStats" data-dfn-for="RTCMediaSourceStats"
+            class="dictionary-members">
+              <dt>
+                <dfn><code>width</code></dfn> of type <span class=
+                "idlMemberType">unsigned long</span>
+              </dt>
+              <dd>
+                <p>
+                  Only applicable to video sources. The width, in pixels, of the
```

Comment:
Given that every single object here has "Only applicable to video sources", should we rename the dictionary "RTCVideoSourceStats"?